crates/typst/src/layout/inline/line.rs

```diff
@@ -76,7 +76,8 @@ impl<'a> Line<'a> {
             .iter()
             .filter_map(Item::text)
             .map(|s| s.stretchability())
-            .sum()
+            // Don’t use sum because it returns -0.0 for an empty iter
+            .fold(Abs::zero(), |acc, x| acc + x)
```

Comment:
I think this comment should go above the `self.items` line. Then you should write "Use fold instead of sum" rather than "Don't use sum".